### PR TITLE
Delete testcase file before every run

### PR DESCRIPTION
**Pull request by everestmz**
cc: @google/clusterfuzz 
Fixes: #129

### CODE REVIEW

File: tool/test/clusterfuzz/testcase_test.py [hunk 1]
```diff
@@ -117,10 +117,24 @@ def test_already_downloaded(self):
       f.write('testcase exists')
     self.assertTrue(os.path.isfile(filename))
 
+    def do_wget(*unused_args):
```
Comment (tanin47) [inline tool/test/clusterfuzz/testcase_test.py]: Could you merge it with the test method below.

File: tool/clusterfuzz/testcase.py [hunk 2]
```diff
@@ -106,8 +107,8 @@ def get_testcase_path(self):
 
     testcase_dir = self.testcase_dir_name()
     filename = os.path.join(testcase_dir, 'testcase%s' % self.file_extension)
-    if os.path.isfile(filename):
-      return filename
+    if os.path.exists(testcase_dir):
```
Comment (tanin47) [inline tool/clusterfuzz/testcase.py]: Make a common's method for delete if exists. Then, we will have only one test below.

**Comment by everestmz** [comment]
Updated!

File: tool/test/clusterfuzz/testcase_test.py [hunk 3]
```diff
@@ -101,27 +101,14 @@ def setUp(self):
...
-    with open(filename, 'w') as f:
-      f.write('testcase exists')
-    self.assertTrue(os.path.isfile(filename))
-
-    result = self.test.get_testcase_path()
-    self.assertEqual(result, filename)
-    self.assert_n_calls(0, [self.mock.get_stored_auth_header,
-                            self.mock.execute])
-
   def test_not_already_downloaded(self):
```
Comment (tanin47) [inline tool/test/clusterfuzz/testcase_test.py]: Rename the test.

**Pull request merged**